File: RecipeClient/recipe/src/app/model/Recipe.ts
import { FoodType } from "./FoodType"

export class Recipe
{
  public id: number
  public userId: number
  public name: string
  public foodType: FoodType
  public ingredients: string
  public description: string

  constructor(id: number, userId: number, name: string, foodType: number, ingredients: string, description: string)
  {
    this.id = id
    this.userId = userId
    this.name = name
    this.foodType = foodType,
    this.ingredients = ingredients,
    this.description = description
  }
}

File: RecipeAPI/RecipeAPI/Models/Recipe.cs
namespace RecipeAPI.Models
{
    public class Recipe
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string Name { get; set; }
        public FoodType FoodType { get; set; }
        public string Ingredients { get; set; }
        public string Description { get; set; }
        public Recipe()
        {

        }

        public Recipe(int userId, string name, FoodType foodType, String ingredients, string description)
        {
            UserId = userId;
            Name = name;
            FoodType = foodType;
            Ingredients = ingredients;
            Description = description;
        }
    }

}
